### PR TITLE
feat(Card): Consume penta tokens

### LINKED ISSUE #9991: Consume core Penta updates: Card
Follow up to core issue https://github.com/patternfly/patternfly/issues/6186

The following was done with core bump PR #10017 

- disabled logic for raised variants in getSelectableModifiers
- removed application of isRounded modifier
- removed application of isFlat modifier
- disabled som unit test
- removed isRounded and isFlat from modifier example
- disabled failing integration test

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #9991 

Codemod issue opened: https://github.com/patternfly/pf-codemods/issues/560

### CODE REVIEW

**Comment by tlabaj** [comment]
@edonehoo I added a new example and a secondary styling switcher to some examples.  Can you please take a look.

**Review by edonehoo — APPROVED** [review]
content looks good to me!

**Review by andrew-ronaldson — APPROVED** [review]
LGTM thanks

**Review by kmcfaul — APPROVED** [review]
lgtm. 

Only thing I noticed between core & react is the menu toggle is using the old dropdown toggle css on core. The border around the menu toggle appearing in this PR should be automatically resolved when the menu toggle penta updates are merged in.

**Comment by srambach** [comment]
It looks like the menu isn't escaping the confines of the expandable/expandable with icon cards - does the menu need to be attached differently?
image

Second thing, the disabled card in https://patternfly-react-pr-10056.surge.sh/components/card#clickable-and-selectable has the wrong text color on the title. In core that's an inline link button, and the correct text color is coming from the styling on a disabled button. Do we want to make a change in core to accommodate using an `<a>` there or should this change to use the button?

**Review by srambach — COMMENTED** [review]
^^ questions above

File: packages/react-core/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -139,24 +104,9 @@ export const Card: React.FunctionComponent<CardProps> = ({
   const containsCardTitleChildRef = React.useRef(false);
 
   const registerTitleId = (id: string) => {
-    setTitleId(id);
     containsCardTitleChildRef.current = !!id;
   };
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Card/Card.tsx:108]: I believe we should be able to remove this `registerTitleId` here and from the context (and its use within CardTitle), as well as the `containsCardTitleChildRef` ref. It was really only being used to get the card title ID from CardTitle to set the aria-labelledby attribute in Card.

In CardTitle we should be able to remove the useEffect on line ~25, too.

**Comment by thatblindgeye** [comment]
@srambach for that bug where the menu isn't escaping the Card container, looks like it's due to the `pf-v5-c-card`'s overflow property that was added? Removing overflow: auto on the card prevents that bug. If we need to have that overflow auto then we could update this particular example to pass to the Dropdown, `popperProps={{appendTo: () => document.body}}`, but may be worth looking into further as a followup for another solution if we can keep the menu appended as close to the trigger as possible

**Review by wise-king-sullyman — APPROVED** [review]
Other than @thatblindgeye's comment above LGTM

**Comment by tlabaj** [comment]
> Second thing, the disabled card in https://patternfly-react-pr-10056.surge.sh/components/card#clickable-and-selectable has the wrong text color on the title. In core that's an inline link button, and the correct text color is coming from the styling on a disabled button. Do we want to make a change in core to accommodate using an `<a>` there or should this change to use the button?

@srambach The consumer is responsible for adding the button to the title.  They may add one one that is an `<a>`. We have controll of that from the react side .  Is there a reason to not support `<a>` in a card title?

**Comment by tlabaj** [comment]
> @srambach for that bug where the menu isn't escaping the Card container, looks like it's due to the `pf-v5-c-card`'s overflow property that was added? Removing overflow: auto on the card prevents that bug. If we need to have that overflow auto then we could update this particular example to pass to the Dropdown, `popperProps={{appendTo: () => document.body}}`, but may be worth looking into further as a followup for another solution if we can keep the menu appended as close to the trigger as possible

@srambach can we look into this on the core side.  We try and append the menu as close to the trigger as possible.

**Comment by srambach** [comment]
I created an issue to [address the title link color on disabled cards](https://github.com/patternfly/patternfly/issues/6307)
And updated the [issue to find a solution for rounded corners](https://github.com/patternfly/patternfly/issues/6045) that clip the right things and not the wrong things... 

✅ CSS

**Pull request merged**